releasenotes/notes/uds-64b11960931d8b77.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add unix domain socket support.
```

Comment:
might be worth calling out how it's enabled (the env var at least) so the release note is more actionable.